test/configCases/node/output-module-external/index.js

```diff
@@ -0,0 +1,10 @@
+import fs from "fs";
```

Comment:
Let's add a test case where we import all Node.js modules (just take them from NodeTargetPlugin and generate a module with require's) just to ensure we can replace `require` on `import` for all Node.js modules (It seems like there was a bug with this once, but I could be wrong, let's check it out.)